bin/src/utils.rs

```diff
@@ -3,6 +3,7 @@ use std::collections::HashMap;
 use lib::{Lint, LINTS};
 use rnix::SyntaxKind;
 
+#[allow(clippy::borrowed_box)]
```

Comment:
Did you look into an alternative?

Reply:
Instead of suppressing the warning with `#[allow(clippy::borrowed_box)]`, a proper fix would require:

1. Changing how individual lints are stored in each module (not using `Box<dyn Lint>`)
2. Modifying the `LINTS` collection to store `&'static dyn Lint` references directly
3. Updating the `lint_map_of` function signature and implementation

However, this would be a significant refactoring that would require updating the entire lint system.

Reply:
maybe something for a separate PR.